src/templates/haml/home5/index.html.haml

```diff
@@ -0,0 +1,450 @@
+- stash(
+-   title => l('Online Trading platform for binary options on Forex, Indices, Commodities and Smart Indices'),
+-   next_link => '/why-us',
+-   createacc => l('Create free account now'),
+- );
+- my $lang = stash->{language};
+%script{:src=>'//use.typekit.net/bfu3ywb.js'}
+:javascript
+  try{Typekit.load();}catch(e){console.log("Typekit load error=",e)}
+  (function(){
+  var throttle =function (callback, limit) {
+      var wait = false;                 // Initially, we're not waiting
+      return function () {              // We return a throttled function
+          if (!wait) {                  // If we're not waiting
+              callback.call();          // Execute users function
+              wait = true;              // Prevent future invocations
+              setTimeout(function () {  // After a period of time
+                  wait = false;         // And allow future invocations
+              }, limit);
+          }
+      }
+  }
+  //var elems=null
+
+  var resetH=function(){
+    var elems=$("#benefits .box-wrap:visible")
+    if(!elems)return
+    var hArr=[]
+    elems.height('')
+    //if($(window).width()<400){return }
+    var gsbH=0
+    var $gridSignupBox=$('.grid-signup-box:visible','#benefits')
+    $gridSignupBox.children().each(function(i,obj){
+      gsbH+=$(obj).outerHeight()
+    })
+    var resizeRow=function(rowObj,nextElIdx,elArr){
+      if(rowObj.rowElArr.length<2 && rowObj.rowElArr[0][0]!=$gridSignupBox[0]){
+      processRow(elArr,nextElIdx)
+      return
+      }
+           var maxH=Math.max.apply(this,rowObj.h)
+            for(var i=0;i<rowObj.rowElArr.length;i++){
+              rowObj.rowElArr[i].outerHeight(maxH)
+            }
+            if(nextElIdx>-1)processRow(elArr,nextElIdx)
+    }
+
+    var processRow=function(elArr,fromElIdx){
+    if(fromElIdx<0)return
+      var $el,top,prevRowTop,currRowObj
+      for(var i=fromElIdx;i<elArr.length;i++){
+        $el=$(elArr[i])
+        top=$el.offset().top.toString()
+        if(prevRowTop!=null && top!=prevRowTop ){
+          resizeRow(currRowObj,i,elArr)
+          return
+        }
+        prevRowTop=top
+        if(!currRowObj)currRowObj={rowElArr:[],h:[]}
+        h=$el.outerHeight()
+        if($el[0]==$gridSignupBox[0]){
+          h=gsbH
+        }
+        currRowObj.h.push(h)
+        currRowObj.rowElArr.push($el)
+        var isLast=i+1==elArr.length
+        if(isLast)resizeRow(currRowObj,-1,elArr)
+      }
+    }
+    processRow(elems,0)
+  }
+  window._waitJq=function(){
+    if(window.$==null){
+      setTimeout(window._waitJq,100)
+      return
+    }else{
+      $(function(){
+        //elems=$("#benefits .box-wrap")
+
+        resetH()
+        $(window).load(function(){resetH()})
+        $(window).resize(resetH)
+        //$(window).resize(throttle(resetH,500))
+      })
+    }
+  }
+  window._waitJq()
+
+  }())
+
+:css
+  .static_full .section:first-child{padding-bottom:0;}
```

Comment:
we generally don't use css inside template but as this is only for ab testing we can keep and it will go away when we remove home5